.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Thank you for submitting a contribution to Apache Metron (Incubating).
+Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions
+
+In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check
+the following:
+
+### For all changes:
+- [ ] Is there a JIRA ticket associated with this PR? 
```

Comment:
In our dev guide it gives directions to create one if a Jira doesn't exist. Do you think it makes sense for these to match?